.github/CONTRIBUTING.md

```diff
@@ -2,4 +2,4 @@
 
 Thanks for considering contributing to Prefect!
 
-To navigate our codebase with confidence, see our [contribution guidelines](https://docs.prefect.io/latest/contributing/overview/).
+To navigate our codebase with confidence, see our [contribution guidelines](/contribute).
```

Comment:
This file is hosted in GitHub and not the docs, so we'll want to keep the full URL:
```suggestion
To navigate our codebase with confidence, see our [contribution guidelines](https://docs.prefect.io/latest/contributing/overview/).
```

Reply:
Good catch!